app/charts/line/lines.tsx

```diff
@@ -122,3 +124,73 @@ const Line = memo(function Line({
 }) {
   return <path d={path} stroke={color} fill="none" />;
 });
+
+const getPointRadius = (dotSize: LineConfig["fields"]["y"]["showDotsSize"]) => {
+  switch (dotSize) {
+    case "Small":
+      return 2;
+    case "Medium":
+      return 3;
+    case "Large":
+      return 4;
+    case undefined:
+      return 2;
+    default:
+      const _check: never = dotSize;
+      return _check;
+  }
+};
+
+export const Points = ({
+  dotSize,
+}: {
+  dotSize: LineConfig["fields"]["y"]["showDotsSize"];
+}) => {
+  const { getX, xScale, getY, yScale, bounds, chartData, getSegment, colors } =
+    useChartState() as LinesState;
+
+  const { margins, chartHeight, width } = bounds;
+
+  const ticksPos = useMemo(() => {
+    return xScale.ticks(bounds.chartWidth / 90).map((tick) => {
+      const x = xScale(tick);
+      const date = xScale.invert(x);
+
+      const bisectDate = bisector(
+        (d: Observation, date: Date) => getX(d).getTime() - date.getTime()
+      ).center;
+
+      const i = bisectDate(chartData, date, 0);
```

Comment:
@bprusinowski this was the guy that was tricking the first value of the data set. I changed the last argument from a 1 to 0 and now all data sets that have a dot on the first index of the data seem to work properly. I should have caught this but unfortunately it seems that the 3 or 4 datasets that I used to  test the feature weren't like this.